caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php

```diff
@@ -413,12 +415,12 @@ public function extra_list_table_actions(array $action_links, EE_Event $event)
                 'EVT_ID' => $event->ID(),
             ];
             $reports_link       = EE_Admin_Page::add_query_args_and_nonce($reports_query_args, REG_ADMIN_URL);
-            $action_links[]     = '<a href="'
-                                  . $reports_link
-                                  . '" title="'
-                                  . esc_attr__('View Report', 'event_espresso')
-                                  . '"><div class="dashicons dashicons-chart-bar"></div></a>'
-                                  . "\n\t";
+            $action_links[]     = '
+                <a href="' . $reports_link . '" 
+                   aria-label="' . esc_attr__('View Report', 'event_espresso') . '"
+                >
+                    <div class="dashicons dashicons-chart-bar"></div>
+                </a>' . "\n\t";
```

Comment:
Do we need "\n\t" in the code?

Reply:
ya i dunno... was tempted to remove it, but don't know why that was added and whether removal will cause issues, so just left it in.

Reply:
not needed, so removed it